week2/README.md

```diff
@@ -0,0 +1,88 @@
+#### Part 1 - Short-answer questions.
+a) Describe the two size classes in iOS.</br>
```

Comment:
Compact Size Class: Often used to describe more constrained spaces (like the width of an iPhone in portrait mode).

Regular Size Class: Denotes more expansive space (like the width of an iPad in both portrait and landscape modes or the width of an iPhone in landscape mode).

Reply:
Ah this makes sense—thanks for the concise and clear definition!